sdk/src/types/block/output/output_id_proof.rs

```diff
@@ -10,18 +10,37 @@ use packable::{Packable, PackableExt};
 #[cfg(feature = "serde")]
 use crate::utils::serde::prefix_hex_bytes;
 use crate::{
-    types::block::{output::Output, slot::SlotIndex, Error},
+    types::block::{output::Output, slot::SlotIndex},
     utils::merkle_hasher::{largest_power_of_two, LEAF_HASH_PREFIX, NODE_HASH_PREFIX},
 };
 
+#[derive(Debug, derive_more::Display)]
+pub enum ProofError {
+    #[display(fmt = "invalid output ID proof kind: {_0}")]
```

Comment:
What's `_0`?

Reply:
It's just what `derive_more` renames the tuple fields to